src/components/tx/TxData.js

```diff
@@ -476,6 +479,9 @@ class TxData extends React.Component {
 
     const renderAccumulatedWeight = () => {
       if (this.props.confirmationData) {
+        if (!this.props.confirmationData.success) {
+          return 'Not available';
+        }
```

Comment:
When decoding an unmined transaction we try to fetch the confirmation data and `success: false` was being returned, instead of showing `NaN` show a use friendly error message.